amundsen_application/static/js/ducks/search/reducer.ts

```diff
@@ -64,25 +72,39 @@ export default function reducer(state: SearchReducerState = initialState, action
       return {
         ...state,
         search_term: action.term,
+        isLoading: true,
```

Comment:
Looks like only the `SearchAll` action will trigger the loading spinner. Do you also want to show the spinner for a `SearchResource` action as well? Generally this will happen when a user clicks on a paginated result. It may or may not be a good user experience.

Reply:
Similar comment here, I see you reset the loading state when `SearchResource` completes/fails but didn't set the loading state to true when `SearchResource` is called. I'd be interested to checkout what this looks and feels like when we use the LoadingSpinner for both cases.

Reply:
updating so searchresource action also sets isLoading to true